pkg/instancetype/instancetype_test.go

```diff
@@ -1379,6 +1386,85 @@ var _ = Describe("Instancetype and Preferences", func() {
 			})
 		})
 
+		Context("Instancetype.Spec.Annotations", func() {
+
+			BeforeEach(func() {
+				instancetypeSpec = &instancetypev1beta1.VirtualMachineInstancetypeSpec{
+					Annotations: make(map[string]string),
+				}
+			})
+
+			DescribeTable("should apply to VMI", func(annotations map[string]string) {
+				instancetypeSpec.Annotations = annotations
+
+				conflicts := instancetypeMethods.Apply(field, instancetypeSpec, nil, &vmi.Spec, &vmi.ObjectMeta)
+				Expect(conflicts).To(BeEmpty())
+				Expect(vmi.ObjectMeta.Annotations).To(Equal(instancetypeSpec.Annotations))
+			},
+				Entry("single annotation", singleAnnotation),
```

Comment:
What's the value in testing against a single annotation? I'd be happy with a single test against multiple tbh.

Reply:
Ok, I will keep just a single tests agains multiple